plugin/pkg/bastion/validator/admission_test.go

```diff
@@ -71,6 +72,11 @@ var _ = Describe("Bastion", func() {
 					SeedName: pointer.String(seedName),
 					Provider: gardencore.Provider{
 						Type: provider,
+						Workers: []gardencore.Worker{
+							{
+								Name: workerName,
```

Comment:
Why is this necessary now?

Reply:
We are now using [ShootEnablesSSHAccess()](https://github.com/gardener/gardener/blob/54b0182e8f7231ca7b5de88bf179fb6dc4192851/pkg/apis/core/helper/helpers.go#L501-L505) which returns `false` when `SSHAccess` is disabled or the `Shoot` is workerless. When the `Shoot` is workerless we accept that it has disabled `ssh` and forbid mutate operations on `Bastion` resources, hence we need to add a worker to the test `Shoot`.